spec/integration/_dev/deploy/tf/spec.yml

```
@@ -25,3 +25,8 @@ spec:
       type: file
       contentMediaType: "application/json"
       required: false
+    - description: Files that can be used with the file provisioner of Terraform
```

Well, seeing in the example that these files could be used also with other terraform resources maybe we shouldn't be so specific.
```suggestion
    - description: Files that can be used with Terraform resources or provisioners
```